superset-frontend/spec/javascripts/dashboard/util/getFilterScopeFromNodesTree_spec.js

```diff
@@ -381,5 +381,96 @@ describe('getFilterScopeFromNodesTree', () => {
         immune: [105, 103, 102, 101, 108, 104],
       });
     });
+
+    // test case 3:
+    // - filter_109
+    // - chart_106
+    // - Tab 1
+    //   - Nested_Tab1
+    //     - chart_101
+    //     - chart_102
+    //   - Nested_Tab2
+    //     - chart_103
+    //     - chart_104
+
+    const nodes3 = [
```

Comment:
this const can live in the test case right? since it's not being used by any other tests

Reply:
yes! fixed.